.github/renovate.json

```diff
@@ -17,13 +17,21 @@
       "customType": "regex",
       "fileMatch": ["\\.mise\\.toml$"],
       "matchStrings": [
-        "^(?<depName>[a-zA-Z0-9_-]+)\\s*=\\s*\"ref:(?<currentValue>.*?)\"$"
+        "(?<depName>[a-zA-Z0-9_-]+)\\s*=\\s*\"ref:(?<currentValue>.*?)\""
```

Comment:
Huomasin että Renovate ajaa regexit koko tiedostoa vasten, ei rivi kerrallaan, joten ankkurit pitää poistaa tästä: https://docs.renovatebot.com/modules/manager/regex/#renovates-regex-differs-from-the-online-tools

Reply:
Kopioitu pullariin
https://github.com/Opetushallitus/koto-rekisteri/pull/546/commits/6693e1665b09a185660d1baceb56a223cff7fb36

Reply:
Tää ei myöskään auttanut. Laitoin tämän sisään pullarissa https://github.com/Opetushallitus/koto-rekisteri/pull/546/ mutta renovate avasi heti uuden pullarin https://github.com/Opetushallitus/koto-rekisteri/pull/550